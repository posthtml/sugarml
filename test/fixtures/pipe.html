p
  | text on its own line 
  span now it's a span!
  |  ah, more text on its own line...
