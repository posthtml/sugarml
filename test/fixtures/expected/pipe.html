<p>text on its own line <span>now it's a span!</span> ah, more text on its own line...</p>
